worlds/hades/Options.py

```diff
@@ -0,0 +1,551 @@
+import typing
+from typing import Dict, Any
+from Options import TextChoice, Option, Range, Toggle, DeathLink, Choice
```

Comment:
Would recommend implementing the `start_inventory_from_pool` option here. To do so, add `StartInventoryPool` to your imports from Options, then in your options dataclass, add the line `start_inventory_from_pool: StartInventoryPool`